packages/shared/src/components/profile/ProfileSettingsMenu.tsx

```diff
@@ -72,6 +74,16 @@ const useMenuItems = (): NavItemProps[] => {
         }
       : undefined;
 
+    const downloadAndroidApp =
+      checkIsBrowser(UserAgent.Android) && !isAndroidApp()
```

Comment:
Is't also an experiment right?

Reply:
@rebelchris I'm actually not entirely sure if this is also part of the experiment or not, since it's in a completely different part of the app. 

I'll ask Product